apicurio-registry/README.md

```diff
@@ -0,0 +1,68 @@
+Apicurio Registry Operator
+===
+
+Requirements
+---
+* Docker
+* [go](https://github.com/golang/go) ( `1.13+`, with `export GO111MODULE='on'`), and `$GOPATH` and `$GOROOT` set. 
+* [Operator SDK](https://github.com/operator-framework/operator-sdk/blob/master/doc/user/install-operator-sdk.md)    
+* A running [OKD](https://www.okd.io/download.html) or [Minishift](https://www.okd.io/minishift/) cluster, 
```

Comment:
So far, this operator can run on vanilla k8s. Running on bare kubernetes allows to be publicly available from k8s operator catalogs and OCP operator catalogs. Unless there is a good reason to use OCP, for reaching a wider audience, keep kubernetes compatible. This means, updating your README and scripts to use `kubectl` instead of `oc`